app.js

```diff
@@ -0,0 +1,139 @@
+  },
+  {
+    id: '4',
+    img: './assets/main/projectdetails.jpg',
+    deskimg: './assets/main/port.jpg',
+    desktech: ['Codekit', 'GitHub', 'Javasript', 'Bootstrap', 'Terminal', 'Codepen'],
+    mobiletech: ['Ruby on rails', 'CSS', 'JavaScript'],
+    live_version: 'https://svitalis123.github.io/portfolio/app',
+    source_link: 'https://github.com/svitalis123/portfolio',
+    head: 'Keeping track of hundreds of components',
+    para: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  },
+  {
+    id: '5',
+    img: './assets/main/projectdetails.jpg',
+    deskimg: './assets/main/port.jpg',
+    head: 'Keeping track of hundreds of components',
+    desktech: ['Codekit', 'GitHub', 'Javasript', 'Bootstrap', 'Terminal', 'Codepen'],
+    mobiletech: ['Ruby on rails', 'CSS', 'JavaScript'],
+    live_version: 'https://svitalis123.github.io/portfolio/app',
+    source_link: 'https://github.com/svitalis123/portfolio',
+    para: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  },
+  {
+    id: '6',
+    img: './assets/main/projectdetails.jpg',
+    deskimg: './assets/main/port.jpg',
+    head: 'Keeping track of hundreds of components',
+    desktech: ['Codekit', 'GitHub', 'Javasript', 'Bootstrap', 'Terminal', 'Codepen'],
+    mobiletech: ['Ruby on rails', 'CSS', 'JavaScript'],
+    live_version: 'https://svitalis123.github.io/portfolio/app',
+    source_link: 'https://github.com/svitalis123/portfolio',
+    para: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  },
+];
+
+function getId(id) {
+  if (window.innerWidth > 920) {
```

Comment:
- [ ] As the previous reviewer mentioned your popup is not centered on the page, kindly make it center so it will be symmetric with the page and look like the Figma design
- [ ] Also you should add a blurred background or overlay, so your design looks like the Figma design

|Your design|Figma design|
|-|-|
|![Screenshot_20220907_214329](https://user-images.githubusercontent.com/25227022/188975515-ac0a7247-c894-4067-8ebb-58b71b0d74ff.png)|![](https://user-images.githubusercontent.com/93039359/188851961-8b6824ca-6fce-4888-946a-92a45aa6336b.png)|